ros/api/v0/hosts.py

```diff
@@ -151,11 +152,12 @@ def build_sort_expression(self, order_how, order_method):
 class HostDetailsApi(Resource):
     profile_fields = {
         'host_id': fields.String(attribute='inventory_id'),
```

Comment:
I am getting key `inventory_id` in response of `/systems` API. Will it make sense to keep `inventory_id` as it is?

Reply:
You mean modifying the key in HostsApi to `host_id` like that in HostDetailsApi ?

Reply:
Sorry for confusion. I think instead of modifying HostsApi, don't rename it here in HostDetailsApi to `host_id`. 
What do you think?

Reply:
Cool, makes sense.